torchaudio/functional/filtering.py

```diff
@@ -884,6 +884,10 @@ def lfilter(
 ) -> Tensor:
     r"""Perform an IIR filter by evaluating difference equation.
 
+    Note:
+        To avoid numerical problems, small filter order or short input sequence is prefered.
+        Using double precision could also minimize numerical precision errors.
```

Comment:
Could I also ask to add a test that demonstrates the numerical precision errors (i.e. higher atol and rtol)? We can then use that test case to track progress. Also, this only applies to the backwards pass as far as I understand?

Reply:
Sure, I will add a test that demonstrate the effect of high order filter, probably filtering an unit impulse and check whether the output value exceed `1` or become `nan`. I also found that the length of input sequence doesn't effect much to the instability.

The numerical problem can happen both to the forward and the backward pass, cuz they share the same filter parameters.